integtests/IntegTests.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
I think you should add the integTest project to the business solution

Reply:
I'm not sure, if I should test the software as a black box, our tests can be completely decoupled for the actual implementation. Don't you think so?

Reply:
There is no impact to be in the same solution